deliverables/README.md

```diff
@@ -32,37 +37,106 @@ What is the role of Javascript in a web page?
 #### meta
 Why do we separate HTML and CSS?  
 
+1. You can apply the same formatting o several HTML elements without rewriting code over and over.  
+2. You can apply appearence and formatting to several HTML pages from a single CSS file.  
+
 #### HTML
 What are classes and IDs (and how are they different)?  
+HTML uses Classes and IDs to refer to CSS.  
+
```

Comment:
Why are the different? Do they refer to CSS, or do they allow us to identify specific elements or groups of elements with our CSS?